style.css

```diff
@@ -0,0 +1,633 @@
+  text-align: start;
+}
+
+.picks-header h3 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 14px;
+  font-weight: bold;
+}
+
+.picks-container {
+  display: flex;
+  align-items: center;
+  padding: 5px;
+  width: 230px;
+  height: 98px;
+}
+
+.picks-container-description {
+  display: flex;
+  text-align: start;
+  justify-content: flex-start;
+  align-items: center;
+  padding-left: 10px;
+}
+
+.picks-container-description a {
+  text-decoration: none;
+  font-size: 14px;
+  color: #333;
+}
+
+.picks-container img {
+  width: 75px;
+  height: 75px;
+}
+
+/* -----------------------------Article Footer----------------------------- */
+
+.article-footer {
+  text-align: start;
+  margin: 10px 0 10px 0;
+  width: 600px;
+  clear: both;
+}
+
+.article-footer-data {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  height: 50%;
+  padding-top: 20px;
+}
+
+.article-footer-btn {
+  width: 55%;
+}
+
+.article-footer-btn button {
+  background-color: #567b95;
+  border: none;
+  border-radius: 3px;
+  color: white;
+  width: 100%;
+  height: 35px;
+  font-weight: bold;
+}
+
+.article-footer-icons {
+  display: flex;
+  justify-content: flex-end;
+  width: 45%;
+}
+
+.article-footer-icons span {
+  font-size: 27px;
+  padding: 0 10px;
+}
+
+.article-footer-icons a {
+  text-decoration: none;
+  color: #837d7d;
+}
+
+.article-footer-title h5,
+.article-footer-title a {
+  color: rgb(138, 134, 134);
+  font-size: 12px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: normal;
+}
+
+/* ---------------------------Recommended Articles------------------------ */
+
+.recommended-articles-containner hr {
+  width: 770px;
+  height: 2px;
+  margin: 0 5px 20px 31px;
+  background-color: black;
+}
+
+.more {
+  display: flex;
+  flex-wrap: wrap;
+  grid-column-start: 1;
+  grid-column-end: 2;
+  grid-row-start: 1;
+  grid-row-end: 2;
+  justify-content: space-evenly;
+}
+
+.more-title {
+  text-align: start;
+  padding-left: 32px;
+}
+
+.more-title a {
+  text-decoration: none;
+  color: black;
+}
+
+.more-title h4 {
+  padding: 0 0 20px 0;
+}
+
+.img-conatiner img {
+  width: 232px;
+  height: 181px;
+}
+
+.img-conatiner h6 {
+  text-align: end;
+  color: gray;
+  font-weight: normal;
+}
+
+.img-description {
+  text-align: start;
+}
+
+.img-description h6 {
+  color: grey;
+  font-weight: normal;
+}
+
+.more-article-container {
+  width: 232px;
+  margin: 5px;
+  height: calc(50% - 10px);
+}
+
+/* -----------------Most popular list---------------------------------- */
+
+.most-popular {
+  grid-column-start: 2;
+  grid-column-end: 3;
+  grid-row-start: 1;
+  grid-row-end: 3;
+}
+
+.most-popular hr {
+  width: 350px;
+  height: 2px;
+  margin: 0 5px 20px 0;
+  background-color: black;
+}
+
+.most-popular-container {
+  text-align: start;
+  padding: 10px;
+  border-top: 1px lightgray solid;
+}
+
+.most-popular h3 {
+  padding: 0 0 20px 0;
+}
+
+.most-popular-container a {
+  text-decoration: none;
+  color: black;
+}
```

Comment:
## No repeating of the same rule[1]

Try this:
```
.color-black{
 color:black;
}
```

In your HTML elements

`<div clas="color-black">`


- [ ] Delete all `color:black;` rule you have in this file and write it the class on your HTML file

###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_